pkgs/development/libraries/gpgme/default.nix

```diff
@@ -9,11 +9,13 @@ let
   gpgProgram = if useGnupg1 then "gpg" else "gpg2";
 in
 stdenv.mkDerivation rec {
-  name = "gpgme-1.8.0";
+  name = "gpgme-1.7.1";
```

Comment:
> Both of these were fixed in gpgme master, and I believe we should better wait for 1.8.1 or 1.9.0 on 1.7.1.

gpgme 1.9.0 is out (#24491) and I guess we can merge it soon (just wasn't sure about `useGnupg1` change). I guess we can merge this shortly after the upgrade :smile:.